DUI3-DX/Directory.Packages.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Serilog.Extensions.Logging" Version="7.0.0" />
```

Comment:
That feels like the wrong version but I could be wrong?

Reply:
It was the version we were using in the connector projects